lib/metamodelutil.js

```diff
@@ -76,12 +76,15 @@ function createNameTable(priorModels, metaModel) {
             }
             table[imp.name] = namespace;
         } else if (imp.$class === `${MetaModelNamespace}.ImportTypes`) {
-            for (const type of imp.types) {
+            const aliasedMap = imp.aliasedTypes ? new Map(imp.aliasedTypes.map(({ name, aliasedName }) => [name, aliasedName]))
+                : new Map();
+            imp.types.forEach((type) => {
+                const localName = aliasedMap.get(type) || type;
                 if (!findDeclaration(modelFile, type)) {
```

Comment:
Should this use `localType`? Instead of `type`?

Reply:
**Note:**
- `localName` is the identifier that refers to the imported type, as it can be aliased.
- 'type' refers to the actual declaration name within the model file that is being imported

So, over here, we are looking to see if the type is present in the model file from where it is imported.